synapse/tests/test_lib_ast.py

```diff
@@ -20,7 +20,8 @@
     'name': 'foo',
     'desc': 'The Foo Module',
     'version': (0, 0, 1),
-    'synapse_minversion': (2, 8, 0),
+    'synapse_minversion': [2, 144, 0],
+    'synapse_version': '>=2.8.0',
```

Comment:
Should these be ">=2.8.0,<3.0.0" since people are likely to copy them? Should we also remove the `synapse_minversion` keys from any tests which are potential examples?

Reply:
Yeah I'll add `<3.0.0` to these, keeping a `synapse_minversion` in there makes sure that the cortex is updated enough to actually check the `synapse_version` requirement.